check_env.py

```diff
@@ -1,13 +1,9 @@
 """ Run this file to check your python installation.
 """
-
-import matplotlib
-matplotlib.use('Qt5Agg')
```

Comment:
The explicit usage of `Qt5Agg` was so we get the same behavior on Windows and macOS. Because `macosx` [comes before other backends](https://github.com/matplotlib/matplotlib/blob/d1567715f1913ce935462c408f02e70e6299b6cb/lib/matplotlib/pyplot.py#L238), it gets picked up first on macOS. 

In the case flagged in #2, I'm not clear why `%matplotlib qt` fails with an import error and `%matplotlib qt5`, but I know why they ultimately fail on Binder: Qt requires a window server and Binder is headless. 

If the class is to be taught entirely with notebooks, then removing the `use()` call is fine.